platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/chatter/protocol/processing/ProcessingTimeSendReceive.java

```diff
@@ -39,18 +39,15 @@ public class ProcessingTimeSendReceive implements MessageProvider, MessageHandle
     /**
      * Creates a new instance.
      *
-     * @param processingTimeInterval
-     * 		the interval at which to send processing time messages
-     * @param time
-     * 		provides a point in time in nanoseconds, should only be used to measure relative time (from one point to
-     * 		another), not absolute time (wall clock time)
-     * @param selfProcessingNanos
-     * 		provides the current value of this node's event processing time in nanoseconds.
+     * @param processingTimeInterval the interval at which to send processing time messages
+     * @param time                   provides a point in time in nanoseconds, should only be used to measure relative
+     *                               time (from one point to another), not absolute time (wall clock time)
+     * @param selfProcessingNanos    provides the current value of this node's event processing time in nanoseconds.
      */
     public ProcessingTimeSendReceive(
             final Time time, final Duration processingTimeInterval, final LongSupplier selfProcessingNanos) {
-        this.processingTimeInterval = processingTimeInterval;
         this.time = time;
+        this.processingTimeInterval = processingTimeInterval;
```

Comment:
Null annotations and checks missing

Reply:
I am not going to add those here, since I only changed the location of this line